pages/users/[id].tsx

```diff
@@ -161,7 +161,9 @@ export default function User({ showNotification, loginContext }: Props) {
 
   const avatarColor = graffitiToColor($user.graffiti)
   const ordinalRank = numberToOrdinal($user.rank)
-  const startDate = new Date(2021, 11, 1)
+  const potentialStart = new Date(2023, 18, 1)
```

Comment:
Added this conditional so that rendering will be successful for our testing pre launch (if we set to `new Date(2023, 18, 1)`, our end date would be before start date, thows error)